evalml/tests/data_checks_tests/test_outliers_data_check.py

```diff
@@ -37,7 +37,7 @@ def test_outliers_data_check_warnings():
             ).to_dict()
         ],
         "errors": [],
-        "actions": [],
+        "actions": [{"code": "DROP_ROWS", "metadata": {"indices": [0, 3, 5, 10]}}],
```

Comment:
Can we add a second outlier in the same row, but different column?  I was going to write up an issue to increase coverage this way, but we could just do it here if you're game!  If not, let me know and I'll press Go on the issue.

Additionally, do we intend to handle that case with multiple outliers in the same row here?  Or keep the duplicate rows in DataCheckAction and let it handle that?

Reply:
That's a great idea! I think we should filter for duplicates, updated impl to handle this case and added a new test :)